airbyte-webapp/src/pages/connections/ConnectionStatusPage/ConnectionStatusPage.module.scss

```diff
@@ -11,24 +11,9 @@
 .actions {
   display: flex;
 
-  .resetButton,
-  .syncButton {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
   .resetButton {
     margin-right: variables.$spacing-md;
```

Comment:
This can be replaced with `gap: variables.$spacing-md` as part of `.actions`

Reply:
Nice suggestion!